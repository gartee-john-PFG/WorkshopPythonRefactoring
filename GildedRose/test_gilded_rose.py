from approvaltests.combination_approvals import verify_all_combinations
from gilded_rose import GildedRose
from item import Item

def test_what_does_update_items_do():

    name = [
        "Default Item",
        "Aged Brie",
        "Backstage passes to a TAFKAL80ETC concert",
        "Sulfuras, Hand of Ragnaros",
    ]
    sell_in = [-1, 0, 1, 5, 6, 10, 11]
    quality = [-1, 0, 1, 2, 48, 49, 50]

    # result = update_item('foo', 0, 0)
    # verify(result, reporter)
    verify_all_combinations(update_item, [name, sell_in, quality])


def update_item(name, sell_in, quality):
    items = [Item(name, sell_in, quality)]
    gilded_rose = GildedRose(items)
    gilded_rose.update_quality()
    result = ""
    for item in gilded_rose.items:
        result += f"name: {item.name} sell_in: {item.sell_in} quality: {item.quality}"
    return result
