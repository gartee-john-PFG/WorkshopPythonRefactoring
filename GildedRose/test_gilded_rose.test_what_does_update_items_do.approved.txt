args: ('Default Item', -1, -1) => 'name: Default Item sell_in: -2 quality: -1'
args: ('Default Item', -1, 0) => 'name: Default Item sell_in: -2 quality: 0'
args: ('Default Item', -1, 1) => 'name: Default Item sell_in: -2 quality: 0'
args: ('Default Item', -1, 2) => 'name: Default Item sell_in: -2 quality: 0'
args: ('Default Item', -1, 48) => 'name: Default Item sell_in: -2 quality: 46'
args: ('Default Item', -1, 49) => 'name: Default Item sell_in: -2 quality: 47'
args: ('Default Item', -1, 50) => 'name: Default Item sell_in: -2 quality: 48'
args: ('Default Item', 0, -1) => 'name: Default Item sell_in: -1 quality: -1'
args: ('Default Item', 0, 0) => 'name: Default Item sell_in: -1 quality: 0'
args: ('Default Item', 0, 1) => 'name: Default Item sell_in: -1 quality: 0'
args: ('Default Item', 0, 2) => 'name: Default Item sell_in: -1 quality: 0'
args: ('Default Item', 0, 48) => 'name: Default Item sell_in: -1 quality: 46'
args: ('Default Item', 0, 49) => 'name: Default Item sell_in: -1 quality: 47'
args: ('Default Item', 0, 50) => 'name: Default Item sell_in: -1 quality: 48'
args: ('Default Item', 1, -1) => 'name: Default Item sell_in: 0 quality: -1'
args: ('Default Item', 1, 0) => 'name: Default Item sell_in: 0 quality: 0'
args: ('Default Item', 1, 1) => 'name: Default Item sell_in: 0 quality: 0'
args: ('Default Item', 1, 2) => 'name: Default Item sell_in: 0 quality: 1'
args: ('Default Item', 1, 48) => 'name: Default Item sell_in: 0 quality: 47'
args: ('Default Item', 1, 49) => 'name: Default Item sell_in: 0 quality: 48'
args: ('Default Item', 1, 50) => 'name: Default Item sell_in: 0 quality: 49'
args: ('Default Item', 5, -1) => 'name: Default Item sell_in: 4 quality: -1'
args: ('Default Item', 5, 0) => 'name: Default Item sell_in: 4 quality: 0'
args: ('Default Item', 5, 1) => 'name: Default Item sell_in: 4 quality: 0'
args: ('Default Item', 5, 2) => 'name: Default Item sell_in: 4 quality: 1'
args: ('Default Item', 5, 48) => 'name: Default Item sell_in: 4 quality: 47'
args: ('Default Item', 5, 49) => 'name: Default Item sell_in: 4 quality: 48'
args: ('Default Item', 5, 50) => 'name: Default Item sell_in: 4 quality: 49'
args: ('Default Item', 6, -1) => 'name: Default Item sell_in: 5 quality: -1'
args: ('Default Item', 6, 0) => 'name: Default Item sell_in: 5 quality: 0'
args: ('Default Item', 6, 1) => 'name: Default Item sell_in: 5 quality: 0'
args: ('Default Item', 6, 2) => 'name: Default Item sell_in: 5 quality: 1'
args: ('Default Item', 6, 48) => 'name: Default Item sell_in: 5 quality: 47'
args: ('Default Item', 6, 49) => 'name: Default Item sell_in: 5 quality: 48'
args: ('Default Item', 6, 50) => 'name: Default Item sell_in: 5 quality: 49'
args: ('Default Item', 10, -1) => 'name: Default Item sell_in: 9 quality: -1'
args: ('Default Item', 10, 0) => 'name: Default Item sell_in: 9 quality: 0'
args: ('Default Item', 10, 1) => 'name: Default Item sell_in: 9 quality: 0'
args: ('Default Item', 10, 2) => 'name: Default Item sell_in: 9 quality: 1'
args: ('Default Item', 10, 48) => 'name: Default Item sell_in: 9 quality: 47'
args: ('Default Item', 10, 49) => 'name: Default Item sell_in: 9 quality: 48'
args: ('Default Item', 10, 50) => 'name: Default Item sell_in: 9 quality: 49'
args: ('Default Item', 11, -1) => 'name: Default Item sell_in: 10 quality: -1'
args: ('Default Item', 11, 0) => 'name: Default Item sell_in: 10 quality: 0'
args: ('Default Item', 11, 1) => 'name: Default Item sell_in: 10 quality: 0'
args: ('Default Item', 11, 2) => 'name: Default Item sell_in: 10 quality: 1'
args: ('Default Item', 11, 48) => 'name: Default Item sell_in: 10 quality: 47'
args: ('Default Item', 11, 49) => 'name: Default Item sell_in: 10 quality: 48'
args: ('Default Item', 11, 50) => 'name: Default Item sell_in: 10 quality: 49'
args: ('Aged Brie', -1, -1) => 'name: Aged Brie sell_in: -2 quality: 1'
args: ('Aged Brie', -1, 0) => 'name: Aged Brie sell_in: -2 quality: 2'
args: ('Aged Brie', -1, 1) => 'name: Aged Brie sell_in: -2 quality: 3'
args: ('Aged Brie', -1, 2) => 'name: Aged Brie sell_in: -2 quality: 4'
args: ('Aged Brie', -1, 48) => 'name: Aged Brie sell_in: -2 quality: 50'
args: ('Aged Brie', -1, 49) => 'name: Aged Brie sell_in: -2 quality: 50'
args: ('Aged Brie', -1, 50) => 'name: Aged Brie sell_in: -2 quality: 50'
args: ('Aged Brie', 0, -1) => 'name: Aged Brie sell_in: -1 quality: 1'
args: ('Aged Brie', 0, 0) => 'name: Aged Brie sell_in: -1 quality: 2'
args: ('Aged Brie', 0, 1) => 'name: Aged Brie sell_in: -1 quality: 3'
args: ('Aged Brie', 0, 2) => 'name: Aged Brie sell_in: -1 quality: 4'
args: ('Aged Brie', 0, 48) => 'name: Aged Brie sell_in: -1 quality: 50'
args: ('Aged Brie', 0, 49) => 'name: Aged Brie sell_in: -1 quality: 50'
args: ('Aged Brie', 0, 50) => 'name: Aged Brie sell_in: -1 quality: 50'
args: ('Aged Brie', 1, -1) => 'name: Aged Brie sell_in: 0 quality: 0'
args: ('Aged Brie', 1, 0) => 'name: Aged Brie sell_in: 0 quality: 1'
args: ('Aged Brie', 1, 1) => 'name: Aged Brie sell_in: 0 quality: 2'
args: ('Aged Brie', 1, 2) => 'name: Aged Brie sell_in: 0 quality: 3'
args: ('Aged Brie', 1, 48) => 'name: Aged Brie sell_in: 0 quality: 49'
args: ('Aged Brie', 1, 49) => 'name: Aged Brie sell_in: 0 quality: 50'
args: ('Aged Brie', 1, 50) => 'name: Aged Brie sell_in: 0 quality: 50'
args: ('Aged Brie', 5, -1) => 'name: Aged Brie sell_in: 4 quality: 0'
args: ('Aged Brie', 5, 0) => 'name: Aged Brie sell_in: 4 quality: 1'
args: ('Aged Brie', 5, 1) => 'name: Aged Brie sell_in: 4 quality: 2'
args: ('Aged Brie', 5, 2) => 'name: Aged Brie sell_in: 4 quality: 3'
args: ('Aged Brie', 5, 48) => 'name: Aged Brie sell_in: 4 quality: 49'
args: ('Aged Brie', 5, 49) => 'name: Aged Brie sell_in: 4 quality: 50'
args: ('Aged Brie', 5, 50) => 'name: Aged Brie sell_in: 4 quality: 50'
args: ('Aged Brie', 6, -1) => 'name: Aged Brie sell_in: 5 quality: 0'
args: ('Aged Brie', 6, 0) => 'name: Aged Brie sell_in: 5 quality: 1'
args: ('Aged Brie', 6, 1) => 'name: Aged Brie sell_in: 5 quality: 2'
args: ('Aged Brie', 6, 2) => 'name: Aged Brie sell_in: 5 quality: 3'
args: ('Aged Brie', 6, 48) => 'name: Aged Brie sell_in: 5 quality: 49'
args: ('Aged Brie', 6, 49) => 'name: Aged Brie sell_in: 5 quality: 50'
args: ('Aged Brie', 6, 50) => 'name: Aged Brie sell_in: 5 quality: 50'
args: ('Aged Brie', 10, -1) => 'name: Aged Brie sell_in: 9 quality: 0'
args: ('Aged Brie', 10, 0) => 'name: Aged Brie sell_in: 9 quality: 1'
args: ('Aged Brie', 10, 1) => 'name: Aged Brie sell_in: 9 quality: 2'
args: ('Aged Brie', 10, 2) => 'name: Aged Brie sell_in: 9 quality: 3'
args: ('Aged Brie', 10, 48) => 'name: Aged Brie sell_in: 9 quality: 49'
args: ('Aged Brie', 10, 49) => 'name: Aged Brie sell_in: 9 quality: 50'
args: ('Aged Brie', 10, 50) => 'name: Aged Brie sell_in: 9 quality: 50'
args: ('Aged Brie', 11, -1) => 'name: Aged Brie sell_in: 10 quality: 0'
args: ('Aged Brie', 11, 0) => 'name: Aged Brie sell_in: 10 quality: 1'
args: ('Aged Brie', 11, 1) => 'name: Aged Brie sell_in: 10 quality: 2'
args: ('Aged Brie', 11, 2) => 'name: Aged Brie sell_in: 10 quality: 3'
args: ('Aged Brie', 11, 48) => 'name: Aged Brie sell_in: 10 quality: 49'
args: ('Aged Brie', 11, 49) => 'name: Aged Brie sell_in: 10 quality: 50'
args: ('Aged Brie', 11, 50) => 'name: Aged Brie sell_in: 10 quality: 50'
args: ('Backstage passes to a TAFKAL80ETC concert', -1, -1) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: -2 quality: 0'
args: ('Backstage passes to a TAFKAL80ETC concert', -1, 0) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: -2 quality: 0'
args: ('Backstage passes to a TAFKAL80ETC concert', -1, 1) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: -2 quality: 0'
args: ('Backstage passes to a TAFKAL80ETC concert', -1, 2) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: -2 quality: 0'
args: ('Backstage passes to a TAFKAL80ETC concert', -1, 48) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: -2 quality: 0'
args: ('Backstage passes to a TAFKAL80ETC concert', -1, 49) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: -2 quality: 0'
args: ('Backstage passes to a TAFKAL80ETC concert', -1, 50) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: -2 quality: 0'
args: ('Backstage passes to a TAFKAL80ETC concert', 0, -1) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: -1 quality: 0'
args: ('Backstage passes to a TAFKAL80ETC concert', 0, 0) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: -1 quality: 0'
args: ('Backstage passes to a TAFKAL80ETC concert', 0, 1) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: -1 quality: 0'
args: ('Backstage passes to a TAFKAL80ETC concert', 0, 2) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: -1 quality: 0'
args: ('Backstage passes to a TAFKAL80ETC concert', 0, 48) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: -1 quality: 0'
args: ('Backstage passes to a TAFKAL80ETC concert', 0, 49) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: -1 quality: 0'
args: ('Backstage passes to a TAFKAL80ETC concert', 0, 50) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: -1 quality: 0'
args: ('Backstage passes to a TAFKAL80ETC concert', 1, -1) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 0 quality: 2'
args: ('Backstage passes to a TAFKAL80ETC concert', 1, 0) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 0 quality: 3'
args: ('Backstage passes to a TAFKAL80ETC concert', 1, 1) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 0 quality: 4'
args: ('Backstage passes to a TAFKAL80ETC concert', 1, 2) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 0 quality: 5'
args: ('Backstage passes to a TAFKAL80ETC concert', 1, 48) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 0 quality: 50'
args: ('Backstage passes to a TAFKAL80ETC concert', 1, 49) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 0 quality: 50'
args: ('Backstage passes to a TAFKAL80ETC concert', 1, 50) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 0 quality: 50'
args: ('Backstage passes to a TAFKAL80ETC concert', 5, -1) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 4 quality: 2'
args: ('Backstage passes to a TAFKAL80ETC concert', 5, 0) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 4 quality: 3'
args: ('Backstage passes to a TAFKAL80ETC concert', 5, 1) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 4 quality: 4'
args: ('Backstage passes to a TAFKAL80ETC concert', 5, 2) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 4 quality: 5'
args: ('Backstage passes to a TAFKAL80ETC concert', 5, 48) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 4 quality: 50'
args: ('Backstage passes to a TAFKAL80ETC concert', 5, 49) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 4 quality: 50'
args: ('Backstage passes to a TAFKAL80ETC concert', 5, 50) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 4 quality: 50'
args: ('Backstage passes to a TAFKAL80ETC concert', 6, -1) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 5 quality: 1'
args: ('Backstage passes to a TAFKAL80ETC concert', 6, 0) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 5 quality: 2'
args: ('Backstage passes to a TAFKAL80ETC concert', 6, 1) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 5 quality: 3'
args: ('Backstage passes to a TAFKAL80ETC concert', 6, 2) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 5 quality: 4'
args: ('Backstage passes to a TAFKAL80ETC concert', 6, 48) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 5 quality: 50'
args: ('Backstage passes to a TAFKAL80ETC concert', 6, 49) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 5 quality: 50'
args: ('Backstage passes to a TAFKAL80ETC concert', 6, 50) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 5 quality: 50'
args: ('Backstage passes to a TAFKAL80ETC concert', 10, -1) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 9 quality: 1'
args: ('Backstage passes to a TAFKAL80ETC concert', 10, 0) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 9 quality: 2'
args: ('Backstage passes to a TAFKAL80ETC concert', 10, 1) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 9 quality: 3'
args: ('Backstage passes to a TAFKAL80ETC concert', 10, 2) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 9 quality: 4'
args: ('Backstage passes to a TAFKAL80ETC concert', 10, 48) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 9 quality: 50'
args: ('Backstage passes to a TAFKAL80ETC concert', 10, 49) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 9 quality: 50'
args: ('Backstage passes to a TAFKAL80ETC concert', 10, 50) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 9 quality: 50'
args: ('Backstage passes to a TAFKAL80ETC concert', 11, -1) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 10 quality: 0'
args: ('Backstage passes to a TAFKAL80ETC concert', 11, 0) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 10 quality: 1'
args: ('Backstage passes to a TAFKAL80ETC concert', 11, 1) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 10 quality: 2'
args: ('Backstage passes to a TAFKAL80ETC concert', 11, 2) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 10 quality: 3'
args: ('Backstage passes to a TAFKAL80ETC concert', 11, 48) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 10 quality: 49'
args: ('Backstage passes to a TAFKAL80ETC concert', 11, 49) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 10 quality: 50'
args: ('Backstage passes to a TAFKAL80ETC concert', 11, 50) => 'name: Backstage passes to a TAFKAL80ETC concert sell_in: 10 quality: 50'
args: ('Sulfuras, Hand of Ragnaros', -1, -1) => 'name: Sulfuras, Hand of Ragnaros sell_in: -1 quality: -1'
args: ('Sulfuras, Hand of Ragnaros', -1, 0) => 'name: Sulfuras, Hand of Ragnaros sell_in: -1 quality: 0'
args: ('Sulfuras, Hand of Ragnaros', -1, 1) => 'name: Sulfuras, Hand of Ragnaros sell_in: -1 quality: 1'
args: ('Sulfuras, Hand of Ragnaros', -1, 2) => 'name: Sulfuras, Hand of Ragnaros sell_in: -1 quality: 2'
args: ('Sulfuras, Hand of Ragnaros', -1, 48) => 'name: Sulfuras, Hand of Ragnaros sell_in: -1 quality: 48'
args: ('Sulfuras, Hand of Ragnaros', -1, 49) => 'name: Sulfuras, Hand of Ragnaros sell_in: -1 quality: 49'
args: ('Sulfuras, Hand of Ragnaros', -1, 50) => 'name: Sulfuras, Hand of Ragnaros sell_in: -1 quality: 50'
args: ('Sulfuras, Hand of Ragnaros', 0, -1) => 'name: Sulfuras, Hand of Ragnaros sell_in: 0 quality: -1'
args: ('Sulfuras, Hand of Ragnaros', 0, 0) => 'name: Sulfuras, Hand of Ragnaros sell_in: 0 quality: 0'
args: ('Sulfuras, Hand of Ragnaros', 0, 1) => 'name: Sulfuras, Hand of Ragnaros sell_in: 0 quality: 1'
args: ('Sulfuras, Hand of Ragnaros', 0, 2) => 'name: Sulfuras, Hand of Ragnaros sell_in: 0 quality: 2'
args: ('Sulfuras, Hand of Ragnaros', 0, 48) => 'name: Sulfuras, Hand of Ragnaros sell_in: 0 quality: 48'
args: ('Sulfuras, Hand of Ragnaros', 0, 49) => 'name: Sulfuras, Hand of Ragnaros sell_in: 0 quality: 49'
args: ('Sulfuras, Hand of Ragnaros', 0, 50) => 'name: Sulfuras, Hand of Ragnaros sell_in: 0 quality: 50'
args: ('Sulfuras, Hand of Ragnaros', 1, -1) => 'name: Sulfuras, Hand of Ragnaros sell_in: 1 quality: -1'
args: ('Sulfuras, Hand of Ragnaros', 1, 0) => 'name: Sulfuras, Hand of Ragnaros sell_in: 1 quality: 0'
args: ('Sulfuras, Hand of Ragnaros', 1, 1) => 'name: Sulfuras, Hand of Ragnaros sell_in: 1 quality: 1'
args: ('Sulfuras, Hand of Ragnaros', 1, 2) => 'name: Sulfuras, Hand of Ragnaros sell_in: 1 quality: 2'
args: ('Sulfuras, Hand of Ragnaros', 1, 48) => 'name: Sulfuras, Hand of Ragnaros sell_in: 1 quality: 48'
args: ('Sulfuras, Hand of Ragnaros', 1, 49) => 'name: Sulfuras, Hand of Ragnaros sell_in: 1 quality: 49'
args: ('Sulfuras, Hand of Ragnaros', 1, 50) => 'name: Sulfuras, Hand of Ragnaros sell_in: 1 quality: 50'
args: ('Sulfuras, Hand of Ragnaros', 5, -1) => 'name: Sulfuras, Hand of Ragnaros sell_in: 5 quality: -1'
args: ('Sulfuras, Hand of Ragnaros', 5, 0) => 'name: Sulfuras, Hand of Ragnaros sell_in: 5 quality: 0'
args: ('Sulfuras, Hand of Ragnaros', 5, 1) => 'name: Sulfuras, Hand of Ragnaros sell_in: 5 quality: 1'
args: ('Sulfuras, Hand of Ragnaros', 5, 2) => 'name: Sulfuras, Hand of Ragnaros sell_in: 5 quality: 2'
args: ('Sulfuras, Hand of Ragnaros', 5, 48) => 'name: Sulfuras, Hand of Ragnaros sell_in: 5 quality: 48'
args: ('Sulfuras, Hand of Ragnaros', 5, 49) => 'name: Sulfuras, Hand of Ragnaros sell_in: 5 quality: 49'
args: ('Sulfuras, Hand of Ragnaros', 5, 50) => 'name: Sulfuras, Hand of Ragnaros sell_in: 5 quality: 50'
args: ('Sulfuras, Hand of Ragnaros', 6, -1) => 'name: Sulfuras, Hand of Ragnaros sell_in: 6 quality: -1'
args: ('Sulfuras, Hand of Ragnaros', 6, 0) => 'name: Sulfuras, Hand of Ragnaros sell_in: 6 quality: 0'
args: ('Sulfuras, Hand of Ragnaros', 6, 1) => 'name: Sulfuras, Hand of Ragnaros sell_in: 6 quality: 1'
args: ('Sulfuras, Hand of Ragnaros', 6, 2) => 'name: Sulfuras, Hand of Ragnaros sell_in: 6 quality: 2'
args: ('Sulfuras, Hand of Ragnaros', 6, 48) => 'name: Sulfuras, Hand of Ragnaros sell_in: 6 quality: 48'
args: ('Sulfuras, Hand of Ragnaros', 6, 49) => 'name: Sulfuras, Hand of Ragnaros sell_in: 6 quality: 49'
args: ('Sulfuras, Hand of Ragnaros', 6, 50) => 'name: Sulfuras, Hand of Ragnaros sell_in: 6 quality: 50'
args: ('Sulfuras, Hand of Ragnaros', 10, -1) => 'name: Sulfuras, Hand of Ragnaros sell_in: 10 quality: -1'
args: ('Sulfuras, Hand of Ragnaros', 10, 0) => 'name: Sulfuras, Hand of Ragnaros sell_in: 10 quality: 0'
args: ('Sulfuras, Hand of Ragnaros', 10, 1) => 'name: Sulfuras, Hand of Ragnaros sell_in: 10 quality: 1'
args: ('Sulfuras, Hand of Ragnaros', 10, 2) => 'name: Sulfuras, Hand of Ragnaros sell_in: 10 quality: 2'
args: ('Sulfuras, Hand of Ragnaros', 10, 48) => 'name: Sulfuras, Hand of Ragnaros sell_in: 10 quality: 48'
args: ('Sulfuras, Hand of Ragnaros', 10, 49) => 'name: Sulfuras, Hand of Ragnaros sell_in: 10 quality: 49'
args: ('Sulfuras, Hand of Ragnaros', 10, 50) => 'name: Sulfuras, Hand of Ragnaros sell_in: 10 quality: 50'
args: ('Sulfuras, Hand of Ragnaros', 11, -1) => 'name: Sulfuras, Hand of Ragnaros sell_in: 11 quality: -1'
args: ('Sulfuras, Hand of Ragnaros', 11, 0) => 'name: Sulfuras, Hand of Ragnaros sell_in: 11 quality: 0'
args: ('Sulfuras, Hand of Ragnaros', 11, 1) => 'name: Sulfuras, Hand of Ragnaros sell_in: 11 quality: 1'
args: ('Sulfuras, Hand of Ragnaros', 11, 2) => 'name: Sulfuras, Hand of Ragnaros sell_in: 11 quality: 2'
args: ('Sulfuras, Hand of Ragnaros', 11, 48) => 'name: Sulfuras, Hand of Ragnaros sell_in: 11 quality: 48'
args: ('Sulfuras, Hand of Ragnaros', 11, 49) => 'name: Sulfuras, Hand of Ragnaros sell_in: 11 quality: 49'
args: ('Sulfuras, Hand of Ragnaros', 11, 50) => 'name: Sulfuras, Hand of Ragnaros sell_in: 11 quality: 50'
